docs/data/sdks/android/ampli.md

```diff
@@ -113,19 +113,19 @@ Initialize Ampli in your code. The `load()` method accepts configuration option
     ```java
     import com.amplitude.ampli.*;
 
-    Ampli.getInstance().load(appContext, new LoadOptions()
-      .setEnvironment(Ampli.Environment.PRODUCTION)
+    Ampli.getInstance().load(this, new LoadOptions()
+      .setClient(new LoadClientOptions().setApiKey(AMPLITUDE_API_KEY))
```

Comment:
The syntax for setting API key in Browser isn't terrible (though still lots of nested objects, etc.) but gets a bit gnarlier in other languages. Wasn't too bad when it was the 1% case but now that it's the 99% case needing all this boilerplate to set the API key is a bummer (as opposed to it being a top-level config option like environment used to be)

Reply:
Good point, I agree that the ampli load in Java specifically looks a bit wordy now, though we have plans to improve the interface near term across the board.